### DISCUSSION TITLE
Hide 'em or kill 'em?

### DISCUSSION THREAD

**Discussion by shcaba** [shcaba/Sebastes_melanops_OR#7] [General]
General consensus is that the kill 'em hypothesis is best supported. Let's use this space to add the support for either hypothesis.

**Comment by aaronmberger-nwfsc** [comment]
Agree that kill 'em is best supported (IMO).  One question that remains if this stands: is there enough evidence/support for the hide 'em hypothesis that it warrants one or more sensitivity runs that are included in the document.
